api/src/common/abstract_tests.py

```diff
@@ -44,6 +44,7 @@ class AbstractAPITestCase:
     factory_kwargs = None
     pop_keys = None
     max_diff = None
+    save_options = []
```

Comment:
I don't think this needs to be inside AbstractAPITestCase. We can probably put this into AbstractSavedTestCase